getting_started/setup_vm/roles/intel/tasks/sgx-driver.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Open Enclave SDK contributors.
```

Comment:
Should we remove this? Or add a CCF copyright header to all of our ansible `.yml`s?

Reply:
I don't think we should remove it, that's where we got it from.

My view is that our ansible files are ancillary configuration, not code, and so they do not need copyright headers.